src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -140,28 +140,33 @@ public void exec(final URI home) throws IOException {
     @Ignore
     @Test
     public void handlesTwoRequestInOneConnection() throws Exception {
-        final String text = "Hello world!";
-        final MkContainer container = new MkGrizzlyContainer().next(
-            new MkAnswer.Simple(
-                Joiner.on(BkBasicTest.CRLF).join(
-                    POST,
-                    HOST,
-                    "Content-Length: 4",
-                    "",
-                    "hi",
-                    "POST / HTTP/1.1",
-                    "Host: localhost",
-                    "Content-Length: 4",
-                    "",
-                    "hi"
-                )
-            )
-        ).start();
-        final URI uri = container.home();
-        final Socket socket = new Socket(uri.getHost(), uri.getPort());
-        new BkBasic(new TkText(text)).accept(socket);
+        final String text = "Hello Twice!";
+        final Socket socket = Mockito.mock(Socket.class);
+        final String requests = Joiner.on(BkBasicTest.CRLF).join(
+            BkBasicTest.POST,
+            BkBasicTest.HOST,
+            "Content-Length: 11",
+            "",
+            "Hello First",
+            BkBasicTest.POST,
+            BkBasicTest.HOST,
+            "Content-Length: 12",
+            "",
+            "Hello Second"
+        );
+        final InputStream input = new ByteArrayInputStream(requests.getBytes());
```

Comment:
@danff let's get rid once used variable
http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#one-time-variables